ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,23 @@
-To help diagnose and fix your issue, please provide as many as
-possible of the following items. Note that issues without any of this
-data may be closed as unreproducable:
+If you are having an issue with GeckoDriver starting up, please make sure that your client bindings support using it.
```

Comment:
When I read this, I'm not completely sure what "client bindings" means. Most people probably use geckodriver via selenium, so I'm assuming you mean "clients (like selenium)".